test_vgpu.sh

```diff
@@ -0,0 +1,216 @@
+    fi
+    set +x
+}
+
+trap on_exit EXIT
+
+error_log="VGPU test failed with the following errors: "
+# Set up internal variables
+_SSH_OPTIONS="\
+    -o BatchMode=yes \
+    -o StrictHostKeyChecking=no \
+    -o UserKnownHostsFile=/dev/null \
+    -i /opt/stack/.ssh/id_rsa"
+
+DOM0_IP=$(grep XENAPI_CONNECTION_URL ${DEVSTACK_PATH}/local.conf | cut -d'=' -f2 | sed 's/[^0-9,.]*//g')
+
+echo "Dom0 IP is: $DOM0_IP"
+
+function on_domzero() {
+    ssh $_SSH_OPTIONS "root@$DOM0_IP" bash -s --
+}
+
+pushd $DEVSTACK_PATH/
+
+./clean.sh
+git checkout master
+git pull --ff origin master
+
+popd
+
+if grep "DEST=" ${DEVSTACK_PATH}/local.conf; then
+    dest=$(grep "DEST=" ${DEVSTACK_PATH}/local.conf | cut -d= -f2)
+    if [ $dest != ${DEVSTACK_PATH} ]; then
+        sed -i '/DEST=*/d' ${DEVSTACK_PATH}/local.conf
+    fi
+fi
+if ! grep "DEST=" ${DEVSTACK_PATH}/local.conf; then
+    sed -i '/[[local|localrc]]/a DEST=/opt/stack/openstack' ${DEVSTACK_PATH}/local.conf
+fi
+
+echo "###################Update openstack repositories#####################"
+PROJECT_LIST=$(echo $ZUUL_CHANGES | tr '^' '\n' | cut -d: -f1 | cut -d/ -f2 | sort | uniq)
+
+pushd ${DEST}
+for dir in ${DEST}/*
+do
+    if [ -d "${dir}" ] ; then
+        echo "check ${dir}"
+        pushd ${dir}
+        if [ -d .git ]; then
+            echo "update repository ${dir}"
+            git checkout master
+            if ! git diff-index --quiet HEAD --; then
+                git checkout *
+            fi
+            git pull --ff origin master
+
+            pkg_flag=${dir##*/}
+            echo $pkg_flag
+            if [[ " ${PROJECT_LIST[@]} " =~ " ${pkg_flag} " ]]; then
+                echo "Fetching change for repository ${dir}"
+                if git rev-parse --quiet --verify $TEST_BRANCH > /dev/null; then
+                    echo "delete the old temp branch"
+                    git branch -D $TEST_BRANCH
+                fi
+                git checkout master -B $TEST_BRANCH
+                git fetch ${FETCH_URL}/openstack/${pkg_flag} $refspec
+                git merge FETCH_HEAD -m "temp branch for vgpu patches merge"
+            fi
+        fi;
+        popd
+    fi
+done
+
+popd
+
+echo "###################Get VGPU type from the host#####################"
+first_vgpu_type=$(on_domzero << END_OF_REQ_VGPU_TYPE
+vgpu_type_list=\$(xe vgpu-type-list --minimal)
+vgpu_type_list=\${vgpu_type_list//,/ }
+for id in \$vgpu_type_list
+do
+    name=\$(xe vgpu-type-param-get uuid=\$id param-name="model-name")
+    if [ "\$name" != "passthrough" ]
+    then
+        echo \$name
+        exit
+    fi
+done
+END_OF_REQ_VGPU_TYPE
+)
+
+echo "###################Devstack start stack#####################"
+pushd $DEVSTACK_PATH/
+if ! grep "enabled_vgpu_types=" ${DEVSTACK_PATH}/local.conf; then
+    echo "[devices]" >> ${DEVSTACK_PATH}/local.conf
+    echo "enabled_vgpu_types = $first_vgpu_type" >> ${DEVSTACK_PATH}/local.conf
+fi
+find $JOURNAL_DIR -name "*.journal" | xargs sudo rm
```

Comment:
ok. But you can use -exec:
`   sudo find $JOURNAL_DIR -name "*.journal" -exec rm {} \;`